WireUI/Sources/WireSidebarUI/Views/SidebarViewControllerDelegate.swift

```diff
@@ -16,9 +16,12 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
+import Foundation
+
 @MainActor
 public protocol SidebarViewControllerDelegate: AnyObject {
     func sidebarViewControllerDidSelectAccountImage(_ viewController: SidebarViewController)
+    func sidebarViewController(_ viewController: SidebarViewController, didTapFoldersAt rect: CGRect)
```

Comment:
`didtapFoldersAt` rect sounds really fragile to me.
Is there no other way to know which folder was selected?

Reply:
@caldrian Maybe this is badly named. This is not about identifying which folder was tapped. It is the frame of the `Folders` button in the sidebar.

Reply:
I'll rename this to make it more explicit so that we know this corresponds to a menu item, not a folder.